docs/reference/ls.md

```diff
@@ -16,8 +16,21 @@ parent="smn_machine_subcmds"
 
     Options:
 
-       --quiet, -q					Enable quiet mode
+       --quiet, -q					                Enable quiet mode
        --filter [--filter option --filter option]	Filter output based on conditions provided
+       --timeout, -t				                Timeout in seconds, default to 10s
+
+## Timeout
+
+The `ls` command tries to reach each host in parallel. If a given host does not answer in less than 10 seconds, the `ls` command
```

Comment:
Pretty good just some small nitpicks on my end.  I'd write:

> The `ls` command tries to reach each host in parallel. If a given host does not answer in less than 10 seconds, the `ls` command will state that this host is in `Timeout` state. In some circumstances (poor connection, high load, or while troubleshooting), you may want to increase or decrease this value. You can use the -t flag for this purpose with a numerical value in seconds.

Also, can you please wrap the lines to 80 characters? (your editor probably has a way to do this automatically) It makes the diffs easier to read on a terminal.